users/users-model.js

```diff
@@ -2,7 +2,8 @@ const db = require('../database/dbConfig');
 
 module.exports = {
   register: (newUser) => {
-    return db('users').insert(newUser);
+    const id = db('users').insert(newUser).then(id => id[0]);
```

Comment:
You could alternatively use [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring) here to shorten this:
```javascript
register: (newUser) => {
  const [ id ] = db('users').insert(newUser).then(ids => ids);
  return db('users').where({ id });
},
```
or, using [async await syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function):
```javascript
register: async (newUser) => {
  const [ id ] = await db('users').insert(newUser);
  return db('users').where({ id });
},
```